scripts/helpers/slashes.js

```diff
@@ -0,0 +1,5 @@
+'use strict'
+
+module.exports = function (str) {
+  return str.replace(/\\/g, '/')
```

Comment:
Should I wrap this in `process.platform === 'win32'`?

Or even use https://www.npmjs.com/package/slash?

Reply:
I tend to add 'process.platform === 'win32', easy and quick.